config/vocab.js

```diff
@@ -8,13 +8,39 @@ module.exports.vocab = {
   rootUrl: 'http://vocabs.ardc.edu.au/repository/api/lda/',
   conceptUri: 'concept.json?_view=all',
   cacheExpiry: 31536000, // one year in seconds
-  providers: {
+  external: {
     geonames: {
       method: 'get',
       url: "http://mint:9001/geonames/search?func=search&q=${query}&format=json",
       options: {
 
       }
     }
+  },
+  party: {
```

Comment:
These queries should be under a property named "queries" and should be accessed using sails.config.vocab.queries[queryName]